script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_1.txt

```diff
@@ -89,28 +89,28 @@ Thinking about it, I've been gazing at this scenery for almost a year and a half
 This is the last time I will see this view on my train ride to school.
 I think back on the past seven years, tracing the landscape as I pass by.
 <Page25>
-When I was ten years old, I recovered from a serious accident that should have killed me instantly.
-I met Sensei.
-Then, I ended up living at the Arima household.
-And so the autumn of my second year of high school has arrived.
+When I was ten years old, I recovered from a serious accident that should have killed me on the spot, //The JP text uses commas instead of periods on this page, I think it flows better with the short sentences.
+Met Sensei,
+Ended up living at the Arima household,
+And finally, began the autumn of my second year of high school.
 <Page26>
-The special event that Sensei told me of when we parted never occurred, and as long as I wear the glasses she gave me, I won't see any of the lines at all.
-I believe that I have been spending my days living a precious, yet ordinary life.
+The special event that Sensei told me of when we parted never occurred, and as long as I wear the glasses she gave me, I won't see any lines at all.
+I believe that I've been spending my days living a precious, yet ordinary life.
 <Page27>
 Starting today, however, things are going to get serious.
-Although I'm only returning to my original home, I'd be stretching it if I said that I felt comfortable about it.
+Although I'm only returning to my original home, I'd be lying if I said that I felt comfortable about it.
 I've gotten so used to an ordinary house that living at a place like the Tohno mansion is going to be a big hassle for me.
 <Page28>
 "...To begin with,
 I can't imagine a Western-style building bigger than my school..."
 I'm amazed at the things I was able to endure as a child.
-The mere thought of returning to such an anachronistic lifestyle makes me feel depressed, even if I don't feel anxious.
+The mere thought of returning to such an anachronistic lifestyle makes me feel depressed, even if I don't feel anxious about it.
```

Comment:
Might be better to reorder this to "Even if I don't feel anxious [about it], the mere thought ... makes me feel depressed."